templates/cluster/openstack-standalone-cp/templates/k0scontrolplane.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: K0sControlPlane
+metadata:
+  name: {{ include "k0scontrolplane.name" . }}
+spec:
+  k0sConfigSpec:
+    args:
+      - --enable-worker
+      - --enable-cloud-provider
+      - --kubelet-extra-args="--cloud-provider=external"
+      - --disable-components=konnectivity-server
+    k0s:
+      apiVersion: k0s.k0sproject.io/v1beta1
+      kind: ClusterConfig
+      metadata:
+        name: k0s
+      spec:
+        api:
+          extraArgs:
+            anonymous-auth: "true"
+        extensions:
+          helm:
+            repositories:
+              - name: openstack
+                url: https://kubernetes.github.io/cloud-provider-openstack/
+            charts:
+              - name: openstack-ccm
+                chartname: openstack/openstack-cloud-controller-manager
+                version: 2.31.1
+                order: 1
+                namespace: kube-system
+                values: |
+                  cloudConfig:
+                    global:
+                      auth-url: {{ .Values.openstack.authURL }}
+                      region: {{ .Values.openstack.region }}
```

Comment:
These make no sense with `secret.create: false`

Reply:
Like I mentioned in the other comment, I wanted to remove this entire block but docs says that these two fields are required. It does make sense that it won't look at this block if we are using an existing secret.

But let me test it by removing them and make sure it works.

Reply:
works.